### DISCUSSION TITLE
Error when uploading a vercel.json file for a cron job.

### DISCUSSION THREAD

**Discussion by Elindo586** [vercel/next.js#74405] [Help] [answered]
### Summary

This is the link of the route.js I want to run for a cron job.... (I haven't tested it yet)
https://github.com/Elindo586/estuff/blob/main/app/my/ecron/route.js

(By the way, how can I test a cron job with the code in a route.js using javascript?.  Usually I need to create a button on  a paje.js to make the route.js run.... but since I am doing a cron job, what would be a good way to test it?)

According to Vercel, I should put a file vercel.json at the root?
https://vercel.com/docs/cron-jobs/quickstart

Well, when I put the vercel.json at the root it simply doesn't upload..  Github tells me there is a vercel error and that is it.

The file I put at the root was this one:
https://github.com/Elindo586/estuff/blob/main/thejsons/vercel.json

Can somebody tell me what is the proper way to setup a cron job?

**Comment by icyJoseph** [comment, marked as answer]
The `vercel.json` file has to go at the root of the project, like, as a sibling to `estuff/package.json`

**Reply by Elindo586** [reply]
Ok.. for some reason it uploaded this time... now lets find out if the code works.. :P
